infer/src/base/Config.ml

```diff
@@ -2111,6 +2111,12 @@ and results_dir =
     ~meta:"dir" "Write results and internal files in the specified directory"
 
 
+and run_as_child =
+  CLOpt.mk_int_opt ~in_help:[] ~long:"run-as-child"
+    "Enable child mode when greater than 0. The integer argument is the identity of the child. \
```

Comment:
The "when greater than 0" part is not true anymore, right?

Reply:
Indeed.